src/mappings/Rewards.ts

```diff
@@ -65,6 +68,8 @@ export async function handleReward(rewardEvent: SubstrateEvent): Promise<void> {
 }
 
 export async function handleSlash(slashEvent: SubstrateEvent): Promise<void> {
+    await handleSlashForAnalytics(slashEvent)
+
     let element = await HistoryElement.get(eventId(slashEvent))
```

Comment:
I think it would be logical to also move the other logic into a separate function like ```handleSlashForTxHistory```